File: Java/Easy/Reverse-Words-in-a-String-III.java
class Solution {
  public String reverseWords(String s) {
    String[] arr = null;
    arr = s.split(" ");
    String answer = "";
    // StringBuilder builder = new StringBuilder();
    for (int i = 0; i < arr.length; i++) {
      StringBuilder builder = new StringBuilder();
      builder.append(arr[i]);
      builder.reverse();
      answer += builder.toString();
      if (i != arr.length - 1) {
        answer += " ";
      }
    }
    return answer;
  }
}

File: Python/Easy/Reverse-Words-in-a-String-III.py
class Solution:
    def reverseWords(self, s: str) -> str:
        sepArr = s.split(' ')
        print(sepArr)
        answer = ""
        for i in range(len(sepArr)):
            answer+= sepArr[i][::-1]
            if (i!=len(sepArr)-1):
                answer+=" "
        return answer
